app/jobs/resolution_proofing_job.rb

```diff
@@ -79,15 +79,15 @@ def make_vendor_proofing_requests(
     should_proof_state_id:,
     ipp_enrollment_in_progress:
   )
-    result = resolution_proofer.proof(
+    result = Proofing::Resolution::ProgressiveProofer.new(
       applicant_pii: applicant_pii,
       user_email: user.confirmed_email_addresses.first.email,
       threatmetrix_session_id: threatmetrix_session_id,
       request_ip: request_ip,
       should_proof_state_id: should_proof_state_id,
       ipp_enrollment_in_progress: ipp_enrollment_in_progress,
       timer: timer,
-    )
+    ).proof
```

Comment:
Are you proposing that we change the proof method to look like so?


```ruby

def proof(
  applicant_pii:,
  request_ip:,
  should_proof_state_id:,
  threatmetrix_session_id:,
  timer:,
  user_email:,
  ipp_enrollment_in_progress:
)
  @applicant_pii = applicant_pii
  @request_ip = request_ip
  @should_proof_state_id = should_proof_state_id
  @threatmetrix_session_id = threatmetrix_session_id
  @timer = timer
  @user_email = user_email
  @ipp_enrollment_in_progress = ipp_enrollment_in_progress
  
  @device_profiling_result = proof_with_threatmetrix_if_needed
  @residential_instant_verify_result = proof_residential_address_if_needed
  @instant_verify_result = proof_id_address_with_lexis_nexis_if_needed
  @state_id_result = proof_id_with_aamva_if_needed

  ResultAdjudicator.new(
    device_profiling_result: device_profiling_result,
    ipp_enrollment_in_progress: ipp_enrollment_in_progress,
    resolution_result: instant_verify_result,
    should_proof_state_id: should_proof_state_id,
    state_id_result: state_id_result,
    residential_resolution_result: residential_instant_verify_result,
    same_address_as_id: applicant_pii[:same_address_as_id],
  )
end
```

I'm fine with the implementation and can update it. I was surprised to see that the progressive proofer method was not stubbed anywhere and assumed it must have been at some point.

Reply:
yup 100% that exactly!

Reply:
> I was surprised to see that the progressive proofer method was not stubbed anywhere and assumed it must have been at some point.

Since it was done as an extraction of existing code, I bet that the the specs kept stubbing/webmocking the things underneath (AAMVA proofer, etc) so stubbing out the whole thing wasn't needed